backend/src/main/java/gov/cdc/usds/simplereport/service/supportescalation/SlackConfigService.java

```diff
@@ -0,0 +1,49 @@
+package gov.cdc.usds.simplereport.service.supportescalation;
+
+import java.util.List;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestClientException;
+import org.springframework.web.client.RestTemplate;
+
+@Service
+@Slf4j
+public class SlackConfigService {
+  private final String escalationMessage = "A new support escalation has been received!";
+
+  private String url = "https://hooks.slack.com/services/";
```

Comment:
since we are hardcoding this here -- should I go into the vault and clean up the `slack-hook-url-dev` that I added for you? let me know if this needs to be done! 😸

Reply:
I got access to my SU and deleted them myself!